src/main/riddl/Venues/Locations/locationTypeDefinitions.riddl

```diff
@@ -0,0 +1,24 @@
+type Status is any of {Draft, Open, Closed} briefly "Internal status of a Location."
+
+type Info is {
+	id: LocationId,
+	name: String,
+  description: String,
+  directions: String briefly "How do you find this location?",
+  latLong: LatLong,
+  url: URL?,
+  capacity: Integer?,
+  accessible: Boolean briefly "Is this location handicapped accessible?",
+  amenities: String*, //TODO: We may need amenities to be searchable - i.e. I need to find a Location that is handicaped accessible and has a kitchen
+  entrances: Address+,
+  isPrivate: Boolean briefly "Is this location accessible outside of the organization owning its venue?"
```

Comment:
To be accessible (or not) outside of the owning organization we would need to have reference to an Organization ID

Reply:
My idea was to go to Venue that owns this Location to make this field effective. Maybe what we need is a reference to the Venue this is linked to?

Reply:
You would need something. Let's think about frequency of use (every time somebody makes a reservation or looks for availability) and what would need to happen to successfully enforce this business rule. Also, I would ask if this feature is important to the Comparison Study? Do we need it now or should it be future scope?